host/src/FunctionsNetHost/Grpc/IncomingGrpcMessageHandler.cs

```diff
@@ -48,6 +49,17 @@ private async Task Process(StreamingMessage msg)
                         responseMessage.FunctionMetadataResponse = BuildFunctionMetadataResponse();
                         break;
                     }
+                case StreamingMessage.ContentOneofCase.WorkerWarmupRequest:
+                    {
+                        Logger.LogTrace("Worker warmup request received.");
+                        Prelauncher.Run();
```

Comment:
I see this called in `Main` also... is it meant to be called twice?

Reply:
Yes, @safihamid suggests we should call it during warmup call as well.